specification/wse/src/main/java/org/kaazing/specification/wse/internal/Functions.java

```diff
@@ -66,6 +66,11 @@ public static String asString(int value) {
         return Integer.toString(value);
     }
 
+    @Function
+    public static String uniqueUri(String origin) {
+        return origin + new String(uniqueId());
```

Comment:
Alternatively, use `wse:append(String, String)` to handle the appending part, then reuse the uniqueId() function as-is.